conanfile.py

```diff
@@ -56,8 +56,10 @@ def build_requirements(self):
 
     def requirements(self):
         self.requires("iomgr/[~=11, include_prerelease=True]@oss/master")
-        self.requires("sisl/[~=11, include_prerelease=True]@oss/master")
-        self.requires("nuraft_mesg/[~=2, include_prerelease=True]@oss/main")
+        #self.requires("sisl/[~=11, include_prerelease=True]@oss/master")
```

Comment:
please ignore this for now, it will be removed once the PR from nuraft_mesg and sisl have been merged.